internal/redfishwrapper/client.go

```diff
@@ -91,11 +104,8 @@ func (c *Client) Open(ctx context.Context) error {
 
 	var err error
 	c.client, err = gofish.ConnectContext(ctx, config)
-	if err != nil {
```

Comment:
Was this intentional to remove the error check?

Reply:
Yep it was intentional, the error check was a superfluous - since its not doing anything with the error there. This was caught by one of the semgrep rules I have setup https://github.com/dgryski/semgrep-go/blob/master/errnilcheck.yml